src/nashpy/algorithms/lemke_howson_lex.py

```diff
@@ -0,0 +1,174 @@
+"""A class for the Lemke Howson algorithm"""
+import warnings
+from itertools import cycle
+
+import numpy as np
+
+from nashpy.integer_pivoting import (
+    make_tableau,
+    non_basic_variables,
+    zero_basic_variables,
+    pivot_tableau_lex,
+)
+
+
+def shift_tableau(tableau, shape):
```

Comment:
I believe this is the same as `lemke_howson.shift_tableau` so let's delete it and import it from there.